src/main/java/me/zeroeightsix/kami/gui/kami/theme/kami/KamiFrameUI.java

```diff
@@ -88,20 +127,19 @@ public void renderComponent(Frame component, FontRenderer fontRenderer) {
             if (component.isMinimized()) {
                 glBegin(GL_LINE_LOOP);
                 {
-                    glVertex2d(component.getWidth() - 15, top_y + 2);
-                    glVertex2d(component.getWidth() - 15, bottom_y + 3);
-                    glVertex2d(component.getWidth() - 10, bottom_y + 3);
-                    glVertex2d(component.getWidth() - 10, top_y + 2);
+                    glVertex2d(component.getWidth() - 12, top_y + 2);
+                    glVertex2d(component.getWidth() - 12, bottom_y + 5);
+                    glVertex2d(component.getWidth() - 7, bottom_y + 5);
+                    glVertex2d(component.getWidth() - 7, top_y + 2);
                 }
-                glEnd();
             } else {
                 glBegin(GL_LINES);
                 {
-                    glVertex2d(component.getWidth() - 15, bottom_y + 4);
-                    glVertex2d(component.getWidth() - 10, bottom_y + 4);
+                    glVertex2d(component.getWidth() - 12, bottom_y + 7.5);
+                    glVertex2d(component.getWidth() - 7, bottom_y + 7.5);
```

Comment:
WHere do these numbers come from?

Reply:
these numbers are the numbers to fix the minimize button clipping out of the frame with the new font size. I can work on trying to make it dynamic if you want